package.json

```diff
@@ -220,7 +220,8 @@
     "tsickle": "^0.34.0",
     "tslint": "^5.16.0",
     "typescript": "~3.2.1",
-    "validate-commit-msg": "^2.14.0"
+    "validate-commit-msg": "^2.14.0",
+    "window-crypto": "^1.1.0"
```

Comment:
Do you need this to use `window.crypto.getRandomValues`?

Reply:
Only for testing. Jest doesn't have a polyfill for `window.crypto` by default. So in order to test it as accurately as possible, I'm just polyfilling the functionality as a whole (vs just mocking it).